index.html

```diff
@@ -17,29 +17,175 @@
       href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap"
       rel="stylesheet"
     />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
   </head>
   <body>
     <header>
       <nav class="navbar">
-        <img class="nav logo" src="images/image_geometry_1.svg" alt="logo" />
-        <img class="nav ic-menu" src="images/ic_menu.svg" alt="ic-menu" />
+        <img
+          class="nav logo icon"
+          src="images/image_geometry_1.svg"
+          alt="logo"
+        />
+        <img class="nav ic-menu icon" src="images/ic_menu.svg" alt="ic-menu" />
       </nav>
     </header>
     <main>
-      <section class="intro">
+      <section class="white-bg">
         <h1 class="heading">Charlie Giorgania</h1>
         <p class="para">
           Hello! I am a software developer! I can help you build a product,
           feature or website. Taka a look of my works. If you like what you see
           and have a project you need coded, don’t hesitate and contact me.
         </p>
-        <button class="btn-normal" type="button">Start collaboration</button>
+        <button class="buttons-primary-dark" type="button">
+          Start collaboration
+        </button>
       </section>
       <img
         class="page-brk"
         src="images/image_geometry_2.svg"
         alt="page break"
       />
+      <section class="works">
+        <h1 class="heading heading-works">Projects</h1>
+        <article class="dark-bg">
+          <img src="images/Rectangle21.png" alt="project image" />
+          <div class="project project1">
+            <h2 class="heading2">Project name goes here</h2>
+            <div class="lang-container">
+              <p class="languages">HTML/CSS</p>
+              <p class="languages">Ruby on Rails</p>
+              <p class="languages">JavaScript</p>
+            </div>
```

Comment:
- [ ]  It will be nice for you to ensure that the list of languages uses a `ul` tag to group them, and `li` per language.
- [ ] Kindly try to change that. And do it for all projects. 😃